src/main/java/org/terasology/segmentedpaths/controllers/PathFollowerSystem.java

```diff
@@ -42,10 +42,24 @@
     @In
     SegmentCacheSystem segmentCacheSystem;
 
+    /**
+     * Returns a tangent to the path at the point the vehicle is currently at.
+     *
+     * @param vehicleEntity Vehicle entity marking a point to which we want a tangent
+     * @return Vector representation of the tangent
+     */
     public Vector3f vehicleTangent(EntityRef vehicleEntity) {
         return vehicleTangent(vehicleEntity, 0, null);
     }
 
+    /**
+     * Returns a tangent to the path at the point the vehicle will be after delta distance.
+     *
+     * @param vehicleEntity Vehicle entity to which we want a tangent
```

Comment:
If you're aligning all the parameter descriptions, do it for _the whole file_. Stay consistent. Also align the return description if you choose to align the params. As far as I know, most of our Javadocs don't align these.

Reply:
I would recommend just not aligning at all